code/SwitchFlitController.php

```diff
@@ -12,6 +12,22 @@ class SwitchFlitController extends \Controller
         'getRecordsForDataObject'
     ];
 
+    private static $key_combo = 'ctrl + 75';
+
+    private static $key_combo_mac = 'meta + 75';
+
+    /**
+     * Provide YAML-specified configuration to the browser
+     * @return string JSON-encoded configuration
+     */
+    public function getConfig()
+    {
+        return (object) [
+            'key_combo' => explode('+', $this->config()->key_combo),
+            'key_combo_mac' => explode('+', $this->config()->key_combo_mac),
+        ];
```

Comment:
To avoid needing to trim once we send this to the frontend, maybe we remove any spaces here?